recipe/meta.yaml

```diff
@@ -107,7 +107,7 @@ outputs:
         ###################################################
         # run-like
         - alembic >=1.13.0
-        - astropy-base
+        - astropy-base ==7.0.0
```

Comment:
Please no. We are really waiting for 7.1 to come out. Please at least do `>=7, <8`.